Makefile

```diff
@@ -5,6 +5,16 @@ lab:
                -p 8888:8888 \
                -v ${PWD}:/home/jovyan/work \
                darribas/gds_dev:5.0
+lablinuxsr:
+	docker run --rm \
+	--user root \
+	-e NB_UID=1001 \
```

Comment:
Could we adopt this in the standard lab? Does this break the lab for others on windows?